### DISCUSSION TITLE
Error while doing npm install

### DISCUSSION THREAD

**Discussion by nandita1** [directus/directus#4785] [Archive]
I'm getting the following error on doing npm install. The version of node I'm using is 14.13.0. Your help will be really appreciated.
![image](https://user-images.githubusercontent.com/36603774/113166895-d13ec680-9260-11eb-87a3-455994a5d440.png)

**Comment by rijkvanzanten** [comment]
Is this a fresh clone?

**Reply by nandita1** [reply]
Yes

**Reply by rijkvanzanten** [reply]
What version of node / npm?

**Reply by rijkvanzanten** [reply]
FWIW, `directory-tree` is listed in both the root `package-lock.json` and the `package.json` for `docs`, so it's not a missing module problem 🤔

**Reply by nandita1** [reply]
Can you give me a full installation procedure required?

**Reply by nandita1** [reply]
I resolved the previous issue but doing npm install in all folders containing package.json s but now I'm getting the following error
![image](https://user-images.githubusercontent.com/36603774/113174305-df441580-9267-11eb-9e88-61765dea1ad7.png)

**Reply by rijkvanzanten** [reply]
> Can you give me a full installation procedure required?

Cloning the repo and running `npm install` in the root of the repo does the trick on my end 👍🏻
